qiita_pet/handlers/study_handlers/artifact.py

```diff
@@ -107,6 +107,16 @@ def patch(self):
         self.write(response)
 
 
+class ArtifactGetSamples(BaseHandler):
+    @authenticated
+    def get(self):
+        aids = map(int, self.request.arguments.get('ids[]', []))
```

Comment:
ohh, got it, worth changing? IMOO no, note that we use self.request.arguments.get in one other place ...